src/oop/oop2.py

```diff
@@ -19,6 +22,14 @@ def __init__(self, num_wheels):
 
 # TODO
 
+class Motorcycle(GroundVehicle):
+  def __init__(self):
+    super().__init__(2)
+
+  @staticmethod
```

Comment:
`@staticmethod` should only be used if only the class can access the method but not the instances themselves.